UPDATING.md

```diff
@@ -35,6 +35,9 @@ assists people when migrating to a new version.
   files for production use cases! While we never really supported
   or should have tried to support docker-compose for production use cases, we now actively
   have taken a stance against supporting it. See the PR for details.
+- [27697](https://github.com/apache/superset/pull/27697) [minor] flask-session bump leads to them
+  deprecating `SESSION_USE_SIGNER`, check your configs as this flag won't do anything moving
+  forward.
```

Comment:
We have this deprecation also:
Deprecate [flask_session.filesystem.FileSystemSessionInterface](https://flask-session.readthedocs.io/en/latest/api.html#flask_session.filesystem.FileSystemSessionInterface) in favor of the broader [flask_session.cachelib.CacheLibSessionInterface](https://flask-session.readthedocs.io/en/latest/api.html#flask_session.cachelib.CacheLibSessionInterface) ([2bc7df](https://github.com/pallets-eco/flask-session/commit/2bc7df1be7b8929e55cb25f13845caf0503630d8)).

seems it will be replaced by `cachelib` so this impacts `SESSION_TYPE`